R/modelling.R

```diff
@@ -295,33 +287,36 @@ get_foi_central_estimates <- function(seromodel_object) {
 #' serodata <- prepare_serodata(chagas2012)
 #' seromodel_object <- run_seromodel(serodata = serodata,
 #'                                   foi_model = "constant")
-#' extract_seromodel_summary(seromodel_object)
+#' extract_seromodel_summary(seromodel_object,
+#'                           serodata = serodata)
 #' @export
-extract_seromodel_summary <- function(seromodel_object) {
+extract_seromodel_summary <- function(seromodel_object, 
+                                      serodata) {
   #------- Loo estimates
-  loo_fit <- loo::loo(seromodel_object$seromodel_fit, save_psis = TRUE, "logLikelihood")
+  loo_fit <- loo::loo(seromodel_object, save_psis = TRUE, "logLikelihood")
```

Comment:
Could you name the argument for "logLikelihood" please? As far as I can tell, this is passed to `...` and it's not entirely clear to where for someone not familiar with the codebase.

Reply:
This argument is making reference to `parameter_name` in [`loo::extract_log_lik()`](https://mc-stan.org/loo/reference/extract_log_lik.html), which refers to the name assign to the log-likelihood in the stan models (see [constant.stan](https://github.com/epiverse-trace/serofoi/blob/dev-return-stanfit/inst/stan/constant.stan), for instance):
```
loo_fit <- loo::loo(seromodel, save_psis = FALSE, pars = c(parameter_name = "logLikelihood"))
```
Since the name `parameter_name` is not very telling, I will add a comment above this line to clarify.